webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/annotation/BratAnnotationEditor.java

```diff
@@ -398,8 +412,14 @@ private SpanAnnotationResponse actionSpan(AjaxRequestTarget aTarget, IRequestPar
     {
         // This is the span the user has marked in the browser in order to create a new slot-filler
         // annotation OR the span of an existing annotation which the user has selected.
-        Offsets userSelectedSpan = getOffsetsFromRequest(request, aCas, aSelectedAnnotation);
+        Optional<Offsets> optUserSelectedSpan = getOffsetsFromRequest(request, aCas,
```

Comment:
How can the code arrive here if no offsets are present?

Reply:
Hmm, there was probably a reason for this once upon a time, but I tested it just now and there seems to be no need for this check. You are right.